input/pagecontent/changes.md

```diff
@@ -27,6 +27,7 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from Immunization.protocolApplied, Immunization.protocolApplied.series, Immunization.protocolApplied.targetDisease, Immunization.protocolApplied.doseNumber[x] [FHIR-44674](https://jira.hl7.org/browse/FHIR-44674), [FHIR-44656](https://jira.hl7.org/browse/FHIR-44656), [FHIR-44654](https://jira.hl7.org/browse/FHIR-44654)
   - removed Must Support from Immunization.reasonCode [FHIR-44654](https://jira.hl7.org/browse/FHIR-44654), [FHIR-45968](https://jira.hl7.org/browse/FHIR-45968)
   - added Must Support to Immunization.primarySource [FHIR-44659](https://jira.hl7.org/browse/FHIR-44659)
+  - changed Immunization.vaccineCode slicing rule from slicing by system to slice by value set [FHIR-46391](https://jira.hl7.org/browse/FHIR-46391)
```

Comment:
Suggest wording change to:
support for AIR codes and AMT Vaccine codes in Immunization.vaccineCode changed from slicing by value:system to value set. These slices are now defined in AU Core and not inherited from AU Base (see related AU Base change [FHIR-44821](https://jira.hl7.org/browse/FHIR-44821)).

Reply:
Fixed